sockeye/inference.py

```diff
@@ -630,6 +648,63 @@ def make_input(sentence_id: int, sentence: str) -> TranslatorInput:
         tokens = list(data_io.get_tokens(sentence))
         return TranslatorInput(id=sentence_id, sentence=sentence.rstrip(), tokens=tokens)
 
+    def batch_translate(self, trans_inputs: List[TranslatorInput]) -> List[TranslatorOutput]:
```

Comment:
would it make the batch_size=1 case less efficient if you collapse `batch_translate()` and `translate`?